package.json

```diff
@@ -505,9 +505,12 @@
     "@types/express": "^4.17.21",
     "cors": "^2.8.5",
     "express": "^4.21.0",
+    "graphql": "^16.12.0",
```

Comment:
I only see use of graphql to import a type - can this be a dev dep?

Reply:
Done.